docs/index.rst

```diff
@@ -11,6 +11,30 @@ Everything else is on
 `Github Wiki <https://github.com/Ulauncher/Ulauncher/wiki/>`_.
 
 
+Launch-Targets
+==============
+
+Additionally to launching Applications, Ulauncher can launch the following targets:
+
+Desktop-Entries
+---------------
+
+Ulauncher indexes desktop-entries in ``~/.local/share/applications`` the same as applications.
```

Comment:
@kstenschke so desktop entries in `~/.local/share/applications` is the same entries in `/usr/share/appplications`, i.e. Application desktop entries. Freedesktop format searches in multiple folders (XDG_DATA_DIRS). Please read https://specifications.freedesktop.org/menu-spec/menu-spec-1.0.html